### ISSUE TITLE
make distclean fails if Ant is not installed

### ISSUE THREAD

**Issue by wvxvw** [opencog/link-grammar#534]
Trying to regenerate `configure` script, I run into this:

``` sh
make[2]: Entering directory '/ubi/wvxvw/projects/link-grammar-fork/bindings/java'
rm -rf .libs _libs
ant clean
make[2]: ant: Command not found
make[2]: *** [Makefile:588: clean-local] Error 127
make[2]: Leaving directory '/ubi/wvxvw/projects/link-grammar-fork/bindings/java'
make[1]: *** [Makefile:435: distclean-recursive] Error 1
make[1]: Leaving directory '/ubi/wvxvw/projects/link-grammar-fork/bindings'
make: *** [Makefile:531: distclean-recursive] Error 1
```

I think `distclean` target needs to be more forgiving--in this case it is most likely that Java code has never been generated, if Ant is absent, so just ignore it, rather than trying to clean.

**Comment by wvxvw** [comment]
Additionally, `make distclean` never succeeds, even if I run `make distclean -k`, the next time I try to run `autogen.sh`, I'm still getting:

``` sh
configure: error: source directory already configured; run "make distclean" there first
```

**Comment by linas** [comment]
This should be fixed in pull req #571 and will be in version 5.4.0 coming out shortly.

I'm closing this bug report, re-open if its an issue.
